app/src/main/res/values-az/strings.xml

```diff
@@ -0,0 +1,346 @@
+<resources>
+    <string name="AppBars_comments">%1$s şərh</string>
+    <string name="AppBars_created">Yaradıldı</string>
+    <string name="AppBars_posts">%1$s elan</string>
+    <string name="AppBars_users_6_months">%1$s istifadəçi / 6 ay</string>
+    <string name="AppBars_users_day">%1$s istifadəçi / gün</string>
+    <string name="AppBars_users_month">%1$s istifadəçi / ay</string>
+    <string name="AppBars_users_week">%1$s istifadəçi/həftə</string>
+    <string name="about_social">İctimai</string>
+    <string name="account_settings_activity_account_settings">Hesab Tənzimləmələri</string>
+    <string name="account_settings_all">Hamısı</string>
+    <string name="account_settings_avatar">Avatar</string>
+    <string name="account_settings_banner">Bayraq</string>
+    <string name="account_settings_bio">Haqqında</string>
+    <string name="account_settings_bot_account">Bot Hesabı</string>
+    <string name="account_settings_default_listing_type">Standart Siyahı Növü</string>
+    <string name="account_settings_default_sort_type">Standart Çeşid Növü</string>
+    <string name="account_settings_display_name">Ekran adı</string>
+    <string name="account_settings_email">E-poçt</string>
+    <string name="account_settings_local">Yerli</string>
+    <string name="account_settings_matrix_user">Matrix İstifadəçi</string>
+    <string name="account_settings_remove_current_avatar">Cari Avatarı sil</string>
+    <string name="account_settings_save_settings">Tənzimləmələri saxla</string>
+    <string name="account_settings_send_notifications_to_email">E-poçta bildirişlər göndər</string>
+    <string name="account_settings_show_avatars">Avatarları göstər</string>
+    <string name="account_settings_show_bot_accounts">Bot hesabların göstər</string>
+    <string name="account_settings_show_notifications_for_new_posts">Yeni elanlar üçün bildirişlər göstər</string>
+    <string name="account_settings_show_nsfw">NSFW Göstər</string>
+    <string name="account_settings_show_read_posts">Oxunan elanları göstər</string>
+    <string name="account_settings_show_scores">Xalları göstər</string>
+    <string name="account_settings_subscribed">Abunə olundu</string>
+    <string name="addBookmark">Əlfəcinlərə əlavə et</string>
+    <string name="app_bars_dot_spacer">·</string>
```

Comment:
Make the original not translatable, and remove all occurrences of this in the other translations.